src/main/java/com/prgrms/be/intermark/domain/ticket/dto/TicketFindResponseDTOs.java

```diff
@@ -0,0 +1,11 @@
+package com.prgrms.be.intermark.domain.ticket.dto;
+
+import lombok.Builder;
+
+import java.util.List;
+
+@Builder
+public record TicketFindResponseDTOs(
+        List<TicketFindResponseDTO> ticketFindResponseDTOs
```

Comment:
```suggestion
        List<TicketFindResponseDTO> tickets
```
컨트롤러에서 반환하던데 이상태면 
```json
{
    "ticketFindResponseDTOs" : [~~~~,~~~~,~~~~]
}
```

가 되니 필드명을 바꿔서 

```json
{
    "tickers" : [~~~,~~~,~~~]
}
```
와 같은 형식으로 내리는게 좋아보입니다
(사실 마르코님 리뷰 사항)